src/main/java/com/google/sps/data/SampleData.java

```diff
@@ -14,9 +14,14 @@
 
 package com.google.sps.data;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Calendar;
+import com.google.gson.Gson;
+
 
 /** Class that stores sample Tutor objects for testing. */
 public final class SampleData {
```

Comment:
Could all of the of the previous methods in sample data like getstudentbyemail or deletestudentbyemail be deleted now that they are not being used anymore?

Reply:
Yes, I think so. I'll make a new commit deleting all the unused methods.

Reply:
I think we still need the getStudentByEmail and getTutorByEmail methods because a few tests still use them, but I'll delete everything else.

Reply:
Cool, sounds good!